lib/app.rb

```diff
@@ -0,0 +1,32 @@
+
+require 'sinatra/base'
+#require 'byebug'
+# require 'pry'
```

Comment:
Jag gnäller på de andra så jag kommer gnälla lite på dig också Bosse. ;-) Försök att undvika att klippa-och-klistra kod utan skriv alltid in koden. Det är ett bättre sätt att lära sig på. Då reflekterar du över vad du skriver och kommer ihåg bättre. Det lyser igenom när du (och de andra) har exakt samma kommentarer och i era filer.